dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -186,7 +187,8 @@ public void testWorkflowDAGCWLMissingTool() throws IOException, TimeoutException
         Assert.assertEquals("JSON should not have any data for nodes and edges", strings.size(),1);
     }
 
-    @Test
```

Comment:
Leave the @Test annotation in place when adding @Ignore, it keeps things a bit more clear.
Also, you can add a comment to @Ignore as a value, something like @Ignore("This test will fail as long as ...")